src/autoonline.cpp

```diff
@@ -132,6 +160,11 @@ void AutoOnline::SetUrl(const std::string &url) {
     sensitive_data_.Set("online_url", url_);
 }
 
+void AutoOnline::SetRemoteScript(const std::string& path) {
+    process_script_ = path;
+    sensitive_data_.Set("process_script", path);
```

Comment:
why is this sensitive_data? It's supposed to be used for stuff like passwords, etc.